### DISCUSSION TITLE
M5stack Camera possible to integrate?

### DISCUSSION THREAD

**Discussion by brot123** [jomjol/AI-on-the-edge-device#982] [General]
Hello, I wanted to ask if anyone has managed to get a m5stack camera running for the project here.
I would like to use the camera because it has a LAN port. Hope someone has ideas? I could imagine to send a Module  to you jomjol if you have time for this.

Greeting from Germany:)

**Comment by jomjol** [comment]
Du kannst einfach versuchenn, es selbst zu kompilieren. Du musst nur in der Plattformio.ini folgendes ändern:

```
[env:esp32cam]
platform = espressif32@4.4.0
;board = esp32cam
board = m5stack-core-esp32
framework = espidf
```

Einfach nur esp32cam mit m5stack-core-esp32 ersetzen. Läuft bei mir durch, kann es aber natürlich nicht testen.

**Comment by brot123** [comment]
Vielen Dank für deine Hilfe, werde ich ausprobieren. Ist damit auch die Ethernet Schnittstelle freigeschalten? Ich stelle mir vor das es nicht so einfach ist da in der firmware nicht integriert oder liege ich da falsch?

**Reply by turbolooser** [reply]
Und schon Erfolg gehabt? Bin auch auf der Suche nach einem besseren Kamerasetup

**Comment by epestela** [comment]
I would also be interested to know if you managed to make it work :)
(also, the PoE module i found does not have an sd card slot, how do you add that? )
